### PR TITLE
Portfolio-Middle-Pr

### PR DESCRIPTION
**Pull request by AtillaTahak**
- Added Works section
- Added the About myself section

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @AtillaTahak, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <div class="work-items-button">
+                <button>css</button>
+                <button>html</button>
+                <button>bootstrap</button>
+                <button>Ruby</button>
+            </div>
```
Comment (mke2111) [inline index.html]: - [x] For the list of technologies, I recommend you not use them as buttons.
- [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+            <div class="work-items-see">
+                <button>See Project</button>
```
Comment (mke2111) [inline index.html]: - [x] For all buttons, follow the button styling i.e Active, Hover and Pressed (see design)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+                  Bootstrap
+                  <br>
+                  Ruby on Rails
+                  <br>
+                  RSPec
+                  <br>
+                  CapyBara
+                  <br>
+                  Selenium
+                </li>
```
Comment (mke2111) [inline index.html]: - [x] For each category, the technology used (e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,241 @@
+/* stylelint-disable no-descending-specificity */
+
+.work {
+  display: grid;
+  grid-template-rows: 1fr;
+  background-color: #e5e5e5;
+  justify-content: center;
+  margin: 0;
+}
```
Comment (mke2111) [inline style.css]: - [x] The section should have the margins (top, bottom, left, right) and uses the font and colors (background and font) defined for the template. 
- [x] There is a missing image for this section.

Your page      ||         Original page
<div class="flex">


</div>


- [x] Add some margin to this section

Your page      ||         Original page
<div class="flex">


</div>

File: style.css [hunk 5]
```diff
@@ -0,0 +1,241 @@
...
+  line-height: 24px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  margin: 24px 0;
+  background-color: #1c1a19;
+  height: 2000px;
+}
```
Comment (mke2111) [inline style.css]: - [x] Change the font color for this section, and center align all the contents.
Your page Vs original
<div class="flex">


</div>

File: index.html [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+                <ul class="work-items-project-button-list">
+                    <li> <a href="#">Html</a> </li>
+                    <li> <a href="#">Booststrap</a> </li>
+                    <li> <a href="#">Ruby</a> </li>
+                    
+                </ul>
+            </div>
+        </div>
+        <div class="seeall">
+            <button>See Project</button>
```
Comment (Alexoid1) [inline index.html:72]: - [x] All the buttons change of style in hover and active state.
Captura de pantalla_2021-09-18_02-04-08

File: index.html [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+            <hr>
+        </div>
+        <div class="work-items">
+            <img src="img/ImgPlaceholder.png" alt="">
+            <h2>
+                Multi-Post Stories
+            </h2>
+            <p class="work-items-p">
+                A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
```
Comment (Alexoid1) [inline index.html:26]: - [x] Kindly add the list of technologies and link to the project just like the Figma design.
Captura de pantalla_2021-09-18_02-21-03

File: index.html [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+        <div class="work-items work-items-project">
```
Comment (Alexoid1) [inline index.html:44]: - [ ] For every project use `font-size, font-family, font-weight` values same as Figma template.
Captura de pantalla_2021-09-18_02-28-52

**Figma**
Captura de pantalla_2021-09-18_02-34-05

File: index.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+            <img class="background" src="./img/Group.png" alt="">
+        </div>
+        <div class="about-items">
+            <hr>
+        </div>
+        <div class="about-items">
+            <ul>
+                <li>Languages
+                    <ul class="sublist">
+                        <li>Javascript</li>
```
Comment (Alexoid1) [inline index.html]: - [ ] Delete bullet points from technologies lists.
Captura de pantalla_2021-09-18_02-37-34

- [ ] For every list use `font-size, font-family, font-weight` values same as Figma template.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
Hi @AtillaTahak ,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:

 For the project to get approved, also implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 8 above)
Comment (AtillaTahak) [reply index.html:44]: font-family doesn't have

Comment (AtillaTahak) [reply index.html:44]: I wish you had looked at the css codes and you would have seen the font-family and font-size specification

File: index.html (hunk 9 above)
Comment (AtillaTahak) [reply index.html]: I wish you had looked at the CSS codes and you would have seen the font family and font-size specification
i deleted now font-family

**Review by uwadonat — APPROVED** [review]
Hi **@Atilla,** 

Your project is complete!  There is nothing else to say other than fine. It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 8 above)
Comment (Alexoid1) [reply index.html:44]: Captura de pantalla_2021-09-18_12-43-23

Comment (Alexoid1) [reply index.html:44]: Yes, I looked at the style.css file and you used another `font-family, font size, font wight` you can check the values in Figma by just clicking on the element, and a right sidebar will display.

Comment (AtillaTahak) [reply index.html:44]: did you before make with website Figma? Font family inter didn't have and if you are write Inter result missing font 
and if you looked at my CSS you must see that inter on CSS :) pls be careful on the review

**Pull request merged**